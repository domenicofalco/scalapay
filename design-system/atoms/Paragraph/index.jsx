import styled from 'styled-components'
import { base } from "fonts";
import { grey900 } from "colors";

export default styled.p`
  font-size: ${base};
  color: ${grey900};
  line-height: 1.3;
`;