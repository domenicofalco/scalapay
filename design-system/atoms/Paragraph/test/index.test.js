import React from "react";
import { cleanup, render } from "@testing-library/react";
import Paragraph from "../";

afterEach(cleanup);

describe("atoms => Paragraph", () => {
  it("It should match snapshot", () => {
    const { asFragment } = render(<Paragraph>Paragraph</Paragraph>);
    expect(asFragment()).toMatchSnapshot();
  });
});
