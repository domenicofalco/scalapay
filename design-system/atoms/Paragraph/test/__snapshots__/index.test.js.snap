// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`atoms => Paragraph It should match snapshot 1`] = `
<DocumentFragment>
  <p
    class="Paragraph-sc-14i1zm6-0 lbtMbQ"
  >
    Paragraph
  </p>
</DocumentFragment>
`;
